lib/benchmark_suite/emailer.rb

```diff
@@ -0,0 +1,24 @@
+require 'rest_client'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.